### PR TITLE
chore(package): installed typescript, node-sass

### PR DESCRIPTION
**Pull request by JiyeonP**
typescript, node-sass 설치했습니다

### CODE REVIEW

**Review by s-yeoyul — APPROVED** [review]
확인했습니다!

File: package.json [hunk 1]
```diff
@@ -6,9 +6,15 @@
     "@testing-library/jest-dom": "^5.15.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "@types/jest": "^27.0.3",
+    "@types/node": "^16.11.10",
+    "@types/react": "^17.0.36",
+    "@types/react-dom": "^17.0.11",
+    "node-sass": "^6.0.1",
```
Comment (woohm402) [inline package.json:13]: https://www.npmjs.com/package/node-sass

> Warning: LibSass and Node Sass are deprecated. While they will continue to receive maintenance releases indefinitely, there are no plans to add additional features or compatibility with any new CSS or Sass features. Projects that still use it should move onto Dart Sass.

Comment (JiyeonP) [reply package.json:13]: 아 그럼 재설치하도록 하겠습니다

**Comment by JiyeonP** [comment]
dart sass로 재설치

**Pull request closed without merging**